gradle.properties

```diff
@@ -20,7 +20,7 @@ group=org.apache.kafka
 #  - tests/kafkatest/__init__.py
 #  - tests/kafkatest/version.py (variable DEV_VERSION)
 #  - kafka-merge-pr.py
-version=5.3.1-ccs-SNAPSHOT
+version=5.3.2-ccs-SNAPSHOT
```

Comment:
Did you verify if we need to update any of?

- tests/kafkatest/__init__.py
- tests/kafkatest/version.py (variable DEV_VERSION)

Reply:
Fixed now, thanks for catching those